notebooks/MAST/downloading_transiting_planet_data/downloading_transiting_planet_data.ipynb

```diff
@@ -0,0 +1,961 @@
+{
```

Comment:
<ul><li>method 1: exo.Mast is your Heading 1.</li><li>exo.MAST is a web service API that is optimized for querying exoplanet data.</li><li>make "exo.MAST documentation" the link and delete "which can be found here." </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>